ci/rpm7/build-dep.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+yum -y install epel-release
+yum -y install \
+  bc \
+  boost-devel \
+  bzip2 \
+  cmake \
+  gcc-c++ \
+  git \
+  gnutls-devel \
+  jq \
+  libgcrypt-devel \
+  libcurl-devel \
+  openssl-devel \
+  libuuid-devel \
+  make \
+  mongodb-org \
+  mongodb-org-shell \
+  nc \
+  pyOpenSSL \
+  python \
+  python-flask \
+  rpm-build \
+  scons \
+  tar
+
+curl -L https://github.com/mongodb/mongo-cxx-driver/archive/legacy-1.1.2.tar.gz | tar xzC /opt/ \
+ && cd /opt/mongo-cxx-driver-legacy-1.1.2 \
+ && scons --disable-warnings-as-errors \
+ && scons install --disable-warnings-as-errors --prefix=/usr/local \
+ && rm -Rf /opt/mongo-cxx-driver-legacy-1.1.2
+
+curl -L https://github.com/miloyip/rapidjson/archive/v1.0.2.tar.gz | tar xzC /opt/ \
+ && mv /opt/rapidjson-1.0.2/include/rapidjson/ /usr/local/include \
+ && rm -Rf /opt/rapidjson-1.0.2
+
+curl -L http://ftp.gnu.org/gnu/libmicrohttpd/libmicrohttpd-0.9.48.tar.gz | tar xzC /opt/ \
+ && cd /opt/libmicrohttpd-0.9.48  \
+ && ./configure --disable-messages --disable-postprocessor --disable-dauth  \
+ && make \
+ && make install \
+ && rm -Rf /opt/libmicrohttpd-0.9.48
+
+curl -L https://nexus.lab.fiware.org/repository/raw/public/storage/gmock-1.5.0.tar.bz2 | tar xjC /opt/ \
+ && cd /opt/gmock-1.5.0 \
+ && ./configure \
+ && make \
+ && make install \
+ && rm -Rf /opt/gmock-1.5.0
+
+curl -L http://mosquitto.org/files/source/mosquitto-1.5.tar.gz | tar xzC /opt/ \
```

Comment:
I understand this should mimic the same procedure for Dockerfile in PR #3194, in particular: https://github.com/telefonicaid/fiware-orion/pull/3194/files#diff-ebacf6f6ae4ee68078bb16454b23247dR39

It seems that the 

```
echo '/usr/local/lib' | tee /etc/ld.so.conf.d/99mosquitto.conf
```

is missing.

(However, this dependecy is not really neded until the MQTT notification work finishes and land into master)

Reply:
I don't think that this operation is needed. This path(/usr/local/lib) already in main path of ld. Let's test.

Reply:
> I don't think that this operation is needed. This path(/usr/local/lib) already in main path of ld. Let's test.

@McMutton , what do you think?

Reply:
In order not blocking this PR due to the pending answer, I'd suggest to comment lines L73 to L77 (adding a comment: `FIXME this will get uncomented once #3001` gets implemented). We don't need this part yet (althought it would be good to have it "at hand" as commented block of code).

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3192/commits/b3b5fece8267bcdc815ba091ff7f2cc626c3cc91